Nodejs/Product/Nodejs/Project/NodeModulesNode.cs

```diff
@@ -339,19 +306,6 @@ private class NpmPathProvider : INpmPathProvider {
                 }
             }
 
-            string message;
-            if (e.WithErrors) {
-                message = SR.GetString(
-                    e.Cancelled ? SR.NpmCancelledWithErrors : SR.NpmCompletedWithErrors,
-                    e.CommandText
-                );
-            } else if (e.Cancelled) {
-                message = SR.GetString(SR.NpmCancelled, e.CommandText);
-            } else {
-                message = SR.GetString(SR.NpmSuccessfullyCompleted, e.CommandText);
-            }
-
-            ForceUpdateStatusBarWithNpmActivitySafe(message);

```

Comment:
Let's keep this logic in here. The primary issue was that we were flooding the status bar with activity *during* the install that wasn't particularly useful. It's useful, however, to report back the final result of the npm install to the status bar.